dhall/src/Dhall/Diff.hs

```diff
@@ -556,6 +556,7 @@ skeleton (Record {}) =
     <>  ignore
     <>  " "
     <>  rbrace
+    <>  " (a record)"
```

Comment:
```suggestion
    <>  " (a record type)"
```

Also, could you add a similar explanation for record literals, too?

Reply:
Oops indeed, thanks you!

Reply:
You're welcome! 🙂